lib/matplotlib/backend_bases.py

```diff
@@ -1734,15 +1728,19 @@ def onHilite(self, ev):
 
             canvas.mpl_connect('motion_notify_event',canvas.onHilite)
         """
+        msg = ("onHilite has been deprecated in 1.5 and will be removed "
+               "in 1.6.  This function has not been used internal by mpl "
+               "sense 2007.")
+        warnings.warn(msg, mplDeprecation)
         if not hasattr(self, '_active'):
             self._active = dict()
 
         under = self.figure.hitlist(ev)
         enter = [a for a in under if a not in self._active]
         leave = [a for a in self._active if a not in under]
-        #print "within:"," ".join([str(x) for x in under])
-        #print "entering:",[str(a) for a in enter]
-        #print "leaving:",[str(a) for a in leave]
+        # print "within:"," ".join([str(x) for x in under])
+        # print "entering:",[str(a) for a in enter]
+        # print "leaving:",[str(a) for a in leave]
```

Comment:
Probably was highlighting them because they aren't valid print statements
anymore. Why not just get rid of them?

On Fri, Jan 9, 2015 at 10:23 AM, Thomas A Caswell <notifications@github.com>
wrote:

> In lib/matplotlib/backend_bases.py
> <https://github.com/matplotlib/matplotlib/pull/3988#discussion-diff-22720945>
> :
>
> >          if not hasattr(self, '_active'):
> >              self._active = dict()
> >
> >          under = self.figure.hitlist(ev)
> >          enter = [a for a in under if a not in self._active]
> >          leave = [a for a in self._active if a not in under]
> > -        #print "within:"," ".join([str(x) for x in under])
> > -        #print "entering:",[str(a) for a in enter]
> > -        #print "leaving:",[str(a) for a in leave]
> > +        # print "within:"," ".join([str(x) for x in under])
> > +        # print "entering:",[str(a) for a in enter]
> > +        # print "leaving:",[str(a) for a in leave]
>
> no, that was intentional (my editor was hilighting them and annoying me)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/3988/files#r22720945>.
>